app/connectors/EnrolmentStoreConnector.scala

```diff
@@ -17,8 +17,8 @@
 package connectors
 
 import javax.inject.{Inject, Named}
-import models.EnrolmentVerifiers
-import play.api.http.Status.{NO_CONTENT, BAD_REQUEST}
+import models.{AwrsUsers, EnrolmentVerifiers}
+import play.api.http.Status._
```

Comment:
Could re-add th removed import with the addition of UK i.e.
`play.api.http.Status.{OK, NO_CONTENT, BAD_REQUEST}`